data/pages-content/faq.md

```diff
@@ -47,20 +49,20 @@ A. The conference will feature around 20 hands-on tutorials & workshops, 120 tal
 
 To get an idea of what to expect, have a look at the [EP2022's programme.](https://ep2022.europython.eu/schedule/)
 
-To select talks, we run a Call for Proposals (CFP). Talks are then curated based on the input from the community voting.
+To select talks, we ran a Call for Proposals (CFP). Talks are then curated based on the input from the community voting.
 
 We are always keen to hear new ideas on how to enrich the programme and other activities you are interested in participating during the conference. Share your thoughts with the programme team at [programme@europython.eu](mailto:programme@europython.eu).
 
 
 ## **Q. When do you accept talk proposals?**
 
-A. The Call for Proposals (CFP) will run for two weeks, plus one week extension between **6-26 March 2023**. We are thrilled to receive proposals on any topic that could be of interest to our community.
+A. The Call for Proposals (CFP) was run between **6-26 March 2023**. We are thrilled to have received more than 500 proposals on a wide range of topics. This record-breaking number of proposals reflected the great width of interests of our community. We are also proud to have provided support to first-time speakers via our [Speaker Mentorship Programme
```

Comment:
I think the URL is not complete in the Speaker mentorship program.

Reply:
What an idiot! Fixed with thanks!